src/io/flutter/logging/FlutterLogFilterPanel.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.uiDesigner.core.GridConstraints;
+import com.intellij.util.ui.UIUtil;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+
+public class FlutterLogFilterPanel {
+  private JPanel root;
+  private JCheckBox chbMatchCase;
+  private JCheckBox chbRegex;
+  private JPanel pnlExpression;
+  private final LogFilterTextField txtExpression;
```

Comment:
nit: could we expand these field names to be less abbreviated?  maybe:

`matchCaseCheckBox`
`regexCheckBox`
`panelExpression` and
`textExpression`

?